.github/workflows/nightly-tests.yml

```diff
@@ -80,7 +80,10 @@ jobs:
       run: cargo build --all --release --all-features
     - name: Run tests
       # Number threads is set to 1 because the runner does not have enough memeory for more.
-      run: PILCOM=$(pwd)/pilcom/ cargo test --all --release --verbose --all-features -- --include-ignored --nocapture --test-threads=1
+      run: cargo test --all --release --verbose --all-features -- --include-ignored --nocapture --test-threads=1
+      env:
+        RUST_LOG: info
```

Comment:
isn't info the default?

Reply:
Default is error.

Reply:
Log level info was also needlessly verbose, leaving the default.